pkg/authorization/admission/restrictusers/restrictusers.go

```diff
@@ -98,7 +99,12 @@ func (q *restrictUsersAdmission) Admit(a admission.Attributes) (err error) {
 	// We only care about rolebindings and policybindings; ignore anything else.
 	gr := a.GetResource().GroupResource()
 	switch {
-	case authorizationapi.IsResourceOrLegacy("policybindings", gr), authorizationapi.IsResourceOrLegacy("rolebindings", gr):
+	case authorizationapi.IsResourceOrLegacy("policybindings", gr):
+		// legacy policy bindings
+	case authorizationapi.IsResourceOrLegacy("rolebindings", gr):
```

Comment:
Since you're proxying through to RBAC rolebindings as the original user, yoiu probably don't care about this case, right?  Admission will run on the RBAC resource.